ansible_collections/arista/avd/docs/plugins/Modules_and_action_plugins/deploy_to_cv.md

```diff
@@ -0,0 +1,119 @@
+---
+# This title is used for search results
+title: arista.avd.deploy_to_cv
+---
+<!--
+  ~ Copyright (c) 2023-2024 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+
+# deploy_to_cv
+
+!!! note
+    Always use the FQCN (Fully Qualified Collection Name) `arista.avd.deploy_to_cv` when using this plugin.
+
+!!! warning "This module is in **preview** mode"
+    This module is not guaranteed to have a backwards compatible interface.
+
+Deploy various objects to CloudVision
+
+## Synopsis
+
+The <code>arista.avd.deploy\_to\_cv</code> module is an Ansible Action Plugin providing the following capabilities\:
+
+\- Verify Devices are in the CloudVision inventory.
+\- Verify Devices are in the Inventory \& Topology Studio.
+\- Update the Device hostname in the Inventory \& Topology Studio as needed.
+\- Create Workspace and build, submit, abandon as needed.
+\- Deploy EOS configurations using \"Static Configlet Studio\".
+\- Create and associate Device and Interface Tags.
+\- Approve, run, cancel Change Controls as needed.
+
+## Parameters
+
+| Argument | Type | Required | Default | Value Restrictions | Description |
+| -------- | ---- | -------- | ------- | ------------------ | ----------- |
+| <samp>cv_servers</samp> | list | True | None |  | List of hostnames or IP addresses for CloudVision instance to deploy to. |
+| <samp>cv_token</samp> | str | True | None |  | Service account token. It is strongly recommended to use Vault for this. |
+| <samp>cv_verify_certs</samp> | bool | optional | True |  | Verifies CloudVison server certificates. |
+| <samp>configuration_dir</samp> | str | True | None |  | Path to directory containing .cfg files with EOS configurations. |
+| <samp>structured_config_dir</samp> | str | True | None |  | Path to directory containing files with AVD structured configurations.
+If found, the <code>serial\_number</code> or <code>system\_mac\_address</code> will be used to identify the Device on CloudVision.
+Any tags found in the structured configuration metadata will be applied to the Device and/or Interfaces. |
+| <samp>structured_config_suffix</samp> | str | optional | yml |  | File suffix for AVD structured configuration files. |
+| <samp>device_list</samp> | list | True | None |  | List of devices to deploy. The names are used to find AVD structured configuration and EOS configuration files. |
```

Comment:
Can be reworded to `The names should match the structured config and EOS config file. `